opm/grid/cpgrid/EntityRep.hpp

```diff
@@ -213,6 +213,7 @@ namespace Dune
         class EntityVariableBase : private std::vector<T>
         {
             friend class CpGridData;
+            using std::vector<T>::operator[];
```

Comment:
Isn't this superfluous?

Reply:
Indeed. it did not work as expected as my compiler always choose the one taking the Entity, anyway.